SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -358,7 +358,10 @@ FLocalDeploymentManager::ERuntimeStartResponse FLocalDeploymentManager::StartLoc
 bool FLocalDeploymentManager::SetupRuntimeFileLogger(const FString& RuntimeLogDir)
 {
 	// Ensure any old log file is cleaned up.
-	RuntimeLogFileHandle.Reset();
+	if (RuntimeLogFileHandle)
+	{
+		RuntimeLogFileHandle.Reset();
```

Comment:
I don't think this fixes an issue - calling Reset on a null handle does nothing so I think this was probably a double delete